### PR TITLE
add minor release tags, update monthly images

### PR DESCRIPTION
**Pull request by jperrin**

### CODE REVIEW

File: library/centos [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+centos7.0.1406: git://github.com/CentOS/sig-cloud-instance-images@30aa31e885e3378203d40e50cfd146b160b589dc docker
+7.0.1406: git://github.com/CentOS/sig-cloud-instance-images@30aa31e885e3378203d40e50cfd146b160b589dc docker
+
+# 6.6
+centos6.6: git://github.com/CentOS/sig-cloud-instance-images@72a13cc9b85c6c500531d5f1e1dd86b8308a1cda docker
+6.6: git://github.com/CentOS/sig-cloud-instance-images@72a13cc9b85c6c500531d5f1e1dd86b8308a1cda docker
+
+# 5.11
+centos5.11 git://github.com/CentOS/sig-cloud-instance-images@e0f32850b2893baaa25efa9d98585f4c92aa83fb docker
+5.11 git://github.com/CentOS/sig-cloud-instance-images@e0f32850b2893baaa25efa9d98585f4c92aa83fb docker
```
Comment (tianon) [inline library/centos]: Looks like you're missing some colons here. :smile:

**Comment by yosifkit** [comment]
LGTM
Build test of #384; 54eef260ad8f2e596c179e42df8a968ea8a45a7c (`centos`):

```console
$ ./bashbrew/bashbrew.sh build https://raw.githubusercontent.com/docker-library/official-images/54eef260ad8f2e596c179e42df8a968ea8a45a7c/library/centos
Fetching centos (git://github.com/CentOS/sig-cloud-instance-images) ...
Processing centos:latest ...
Processing centos:centos7 ...
Processing centos:7 ...
Processing centos:centos6 ...
Processing centos:6 ...
Processing centos:centos5 ...
Processing centos:5 ...
Processing centos:centos7.0.1406 ...
Processing centos:7.0.1406 ...
Processing centos:centos6.6 ...
Processing centos:6.6 ...
Processing centos:centos5.11 ...
Processing centos:5.11 ...
```

**Comment by yosifkit** [comment]
ping @tianon

**Comment by tianon** [comment]
LGTM

**Pull request merged**